### ISSUE TITLE
Очень досадный конфликт

### ISSUE THREAD

**Issue by bbslipers** [ArtFect/Smooth-Zoom#3]
Совершенно случайно было обнаружено, что данный мод конфликтует с модом Chisels & Bits.
В Chisels & Bits перестает работать функция "Нарисованная область".
Если можно исправьте пожалуйста.
В остальном мод прекрасен.

**Comment by ArtFect** [comment]
А как она конкретно перестает работать? Можно гиф, видео или попдробнее описание?
Пытался проверить и у меня вроде все ок или я что-то делаю не так. Не особо знаком с этим модом
https://youtu.be/iWXOJ-Gue1s

**Comment by bbslipers** [comment]
Нет, Вы все делаете верно. У меня после выделения области просто не ставятся мини-блоки. Выделение области пропадает и ни чего не происходит. Вижу, что у Вас все работает корректно. Если не сложно напишите версии форжа и модов.

**Comment by bbslipers** [comment]
Если оставить только Smooth-Zoom и Chisels & Bits все работает. Продолжу исследование видимо какой-то еще мод дает такой совокупный эффект.

**Comment by bbslipers** [comment]
Методом эксперимента выявили, что функциональность страдает при переименовании файла мода.
У меня в сборке все клиентские моды переименованы, что бы было удобнее ориентироваться - так например файл с данной модификацией назывался _cln_smoothzoom-1.12.2-1.0.jar и при таком имени обнаруживалась описанная ситуация, стоит вернуть оригинальное имя Smooth Zoom-1.0.jar как все чудесным образом начинает работать ))) Вот такая вот заковыка )))

**Comment by bbslipers** [comment]
Так и есть smoothzoom-1.12.2-1.0.jar - допустимое имя, но как только впереди ставлю нижнее подчеркивание (_smoothzoom-1.12.2-1.0.jar) перестает работать. Чудеса )))
